modules/claims_api/app/controllers/concerns/claims_api/dependent_claimant_verification.rb

```diff
@@ -10,16 +11,22 @@ module DependentClaimantVerification
       def validate_dependent_by_participant_id!(participant_id, dependent_first_name, dependent_last_name)
         return if valid_participant_dependent_combo?(participant_id, dependent_first_name, dependent_last_name)
 
-        raise ::Common::Exceptions::InvalidFieldValue.new('participant_id: dependent name combo',
-                                                          "#{participant_id}: #{dependent_first_name} " \
-                                                          "#{dependent_last_name}")
+        detail = 'The claimant is not listed as a dependent for the specified Veteran. Please submit VA Form 21-686c ' \
+                 'to add this dependent.'
+        raise ::Common::Exceptions::UnprocessableEntity.new(detail:)
+      end
+
+      def validate_poa_code_by_participant_id!(participant_id, poa_code)
+        return if valid_participant_poa_combo?(participant_id, poa_code)
+
+        raise ::Common::Exceptions::UnprocessableEntity.new(detail: 'The requested POA code could not be found.')
       end
     end
 
     private
 
-    def normalize_name(name)
-      name.to_s.strip.upcase
+    def normalize(item)
+      item.to_s.strip.upcase
```

Comment:
Some POA codes have letters (e.g., `00A`). They are all caps from what I can tell. Assuming case does not matter, but let me know if it does.